config/default.yml

```diff
@@ -30,6 +30,7 @@ AllCops:
     - '**/*.rbx'
     - '**/*.ru'
     - '**/*.ruby'
+    - '**/*.schema'
```

Comment:
Note, I'm worried about triggering parse errors when the generic extension name `*.schema` is used in products other than Ridgepole. Expect no conflicts.

Reply:
Indeed it is. I searched on GitHub, but I don't see anything other than Ridgepole that uses `*.schema` that we would use in the same products. Please point out if I have missed something.
https://github.com/search?q=path%3A*.schema&type=code